harness/determined/common/api/bindings.py

```diff
@@ -9593,7 +9593,7 @@ class v1TaskLogsFieldsResponse:
     agentIds: "typing.Optional[typing.Sequence[str]]" = None
     allocationIds: "typing.Optional[typing.Sequence[str]]" = None
     containerIds: "typing.Optional[typing.Sequence[str]]" = None
-    rankIds: "typing.Optional[typing.Sequence[int]]" = None
```

Comment:
rank ID _is_ an int. changing the API to a string to accomplish this seems like a strange choice